FreeRTOS/Demo/Posix_GCC/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.15)
+
+project( posix_demo )
+
+set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_WINDOWS_")
```

Comment:
If this is for a POSIX demo is there a reason we're adding the WINDOWS flag?

Reply:
This macro is used in TimerDemo.c when the timer is not real real time.
https://github.com/FreeRTOS/FreeRTOS/blob/1277ba1661163eba15484051778c86a72a05e432/FreeRTOS/Demo/Common/Minimal/TimerDemo.c#L865-L878
It is also a mapping with existing Makefile.
https://github.com/FreeRTOS/FreeRTOS/blob/1277ba1661163eba15484051778c86a72a05e432/FreeRTOS/Demo/Posix_GCC/Makefile#L66